src/Views/Comparator.elm

```diff
@@ -195,16 +247,49 @@ foodComparatorView { session } { comparisonUnit, switchComparisonUnit } =
             Ok chartsData ->
                 div [ class "h-100" ]
                     [ node "chart-food-comparator"
-                        [ chartsData
-                            |> Encode.list
-                                (\( name, entries ) ->
-                                    Encode.object
-                                        [ ( "label", Encode.string name )
-                                        , ( "data", Encode.list Impact.encodeAggregatedScoreChartEntry entries )
-                                        ]
-                                )
-                            |> Encode.encode 0
-                            |> attribute "data"
+                        [ if not groupByProtectionAreas then
+                            chartsData
+                                |> List.map
+                                    (Tuple.mapSecond
+                                        (Impact.getAggregatedScoreData builderDb.impacts .ecoscoreData
+                                            >> List.sortWith labelComparison
+                                            >> List.reverse
+                                        )
+                                    )
+                                |> Encode.list
+                                    (\( name, entries ) ->
+                                        Encode.object
+                                            [ ( "label", Encode.string name )
+                                            , ( "data", Encode.list Impact.encodeAggregatedScoreChartEntry entries )
+                                            ]
+                                    )
+                                |> Encode.encode 0
+                                |> attribute "data"
+
+                          else
+                            chartsData
+                                |> List.map
+                                    (Tuple.mapSecond
+                                        (Impact.toProtectionAreas builderDb.impacts
+                                            >> (\protectionAreas ->
+                                                    [ { name = "Climat", color = "#7f7f7f", value = Unit.impactToFloat protectionAreas.climate }
+                                                    , { name = "Biodiversité", color = "#00b050", value = Unit.impactToFloat protectionAreas.biodiversity }
+                                                    , { name = "Santé environnementale", color = "#ffc000", value = Unit.impactToFloat protectionAreas.health }
+                                                    , { name = "Ressource", color = "#0070c0", value = Unit.impactToFloat protectionAreas.resources }
+                                                    ]
+                                                        |> List.reverse
+                                               )
```

Comment:
Niot: how about

```elm
    >> (\{ climate, biodiversity, health, resources } ->
            List.reverse
                [ { name = "Climat", color = "#7f7f7f", value = Unit.impactToFloat climate }
                , { name = "Biodiversité", color = "#00b050", value = Unit.impactToFloat biodiversity }
                , { name = "Santé environnementale", color = "#ffc000", value = Unit.impactToFloat health }
                , { name = "Ressource", color = "#0070c0", value = Unit.impactToFloat resources }
                ]
        )
```

Reply:
Also, why the need of reversing the list?

Reply:
the list needs to be reverted because if i understand correctly the stacked data are presented with the first data point to the rightmost, and we'd like it to the leftmost. It's done the same for the "non-grouped" version